components/financial_assistance/app/models/financial_assistance/application.rb

```diff
@@ -128,6 +129,7 @@ class Application # rubocop:disable Metrics/ClassLength TODO: Remove this
     embeds_many :relationships, inverse_of: :application, class_name: '::FinancialAssistance::Relationship', cascade_callbacks: true, validate: true
     embeds_many :applicants, inverse_of: :application, class_name: '::FinancialAssistance::Applicant', cascade_callbacks: true, validate: true
     embeds_many :workflow_state_transitions, class_name: "WorkflowStateTransition", as: :transitional
+    has_many :transactions, as: :transactable, class_name: '::Transmittable::Transaction'
```

Comment:
@ymhari As discussed,
- This is redundant. Please remove this line.
- Add tests to verify the association of an Application with the transactions.

Reply:
Thanks for the catch.